doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1017,8 +1027,8 @@ Interpretation scopes used in the standard library of Coq
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 We give an overview of the scopes used in the standard library of Coq.
-For a complete list of notations in each scope, use the commands Print
-Scopes or Print Scope scope.
+For a complete list of notations in each scope, use the commands ``Print
+Scopes`` or ``Print Scope scope``.
```

Comment:
You could make this ```:cmd:`Print Scopes` or :cmd:`Print Scope` ``` by first adding indexes to these command "variants" (variants do not get an index by default). Their definitions are just a bit below and to add an index, use option `:name:`. Note that I don't really understand why both are marked as variants, probably one of them shouldn't.

If you do this change, you can also reindent the variants inside the block defined by the main command...